<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MW0</class>
 <widget class="QMainWindow" name="MW0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LabWork</string>
  </property>
  <widget class="QWidget" name="CW01">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>731</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить данные из введённого адреса</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите город, улицу, номер улицы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>г. Чита, ул. Баргузинская, 49а</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>731</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть выбранное место на статической карте</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Спутниковый снимок?</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть выбранное место как картинку с карты</string>
    </property>
   </widget>
  </widget>
  <action name="action_8_ZIP_JSON">
   <property name="text">
    <string>ЛР-8 ZIP и JSON</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
